modules/dxp/apps/commerce-avalara/commerce-avalara-tax-engine-fixed-web/src/main/java/com/liferay/commerce/avalara/tax/engine/fixed/web/internal/servlet/taglib/ui/CommerceTaxMethodAvalaraScreenNavigationCategory.java

```diff
@@ -137,6 +165,38 @@ public void render(
 			"/avalara_settings.jsp");
 	}
 
+	private void _setCompanies(HttpServletRequest httpServletRequest) {
+		Map<String, String> companies = new HashMap<>();
+
+		try {
+			companies = _commerceAvalaraConnector.queryCompanies();
+		}
+		catch (Exception exception) {
+		}
+
+		httpServletRequest.setAttribute("avalaraCompanies", companies);
+	}
+
+	private boolean _verifyConnection(
+		CommerceAvalaraConnectorConfiguration
+			commerceAvalaraConnectorConfiguration) {
+
```

Comment:
I think it's more readable to pass the companyId to _verifyConnection and get the configuration from inside the _verify

Reply:
Agreed, very nice find! 😄